### PR TITLE
#580: Check on the address country code

### PR DESCRIPTION
**Pull request by hani-ibrahim**
fixing #580

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/AddressBookImportDataModelCreationStrategy.swift [hunk 1]
```diff
@@ -31,6 +31,13 @@ extension AddressBookImportDataModelCreationStrategy: CNContactPickerDelegate {
     func contactPicker(_ picker: CNContactPickerViewController, didSelect contactProperty: CNContactProperty) {
         picker.dismiss(animated: true) { [weak self] in
             guard let strongSelf = self else { return }
+            guard let postalAddress = contactProperty.value as? CNPostalAddress,
+                postalAddress.isoCountryCode.lowercased() == AtlasAPIClient.shared?.salesChannelCountry.lowercased()
```
Comment (bonkey) [inline AtlasUI/AtlasUI/AddressBookImportDataModelCreationStrategy.swift]: Maybe a function somewhere (in AtlasAPIClient most probably) to have this check there?

Comment (bonkey) [inline AtlasUI/AtlasUI/AddressBookImportDataModelCreationStrategy.swift]: Or maybe `init throws` in `AddressFormDataModel`?

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**